test/entrypoint/srv_select.lua

```diff
@@ -192,6 +192,34 @@ package.preload['customers-storage'] = function()
     }
 end
 
+package.preload['statistics-storage'] = function()
+    return {
+        role_name = 'statistics-storage',
+        dependencies = {'cartridge.roles.crud-storage'},
+        init = function()
```

Comment:
Could you define function in helpers to allow reuse in other entrypoints? I want to use it in PR #181, but my tests uses another entrypoint (`srv_ddl.lua`).

Reply:
I like the idea, will look how to better do so.